src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/patientenaufenthalt/VersorgungsaufenthaltAdminEntryConverter.java

```diff
@@ -0,0 +1,103 @@
+package org.ehrbase.fhirbridge.ehr.converter.specific.patientenaufenthalt;
+
+import org.ehrbase.fhirbridge.ehr.converter.generic.EntryEntityConverter;
+import org.ehrbase.fhirbridge.ehr.converter.generic.TimeConverter;
+import org.ehrbase.fhirbridge.ehr.opt.patientenaufenthaltcomposition.definition.VersorgungsaufenthaltAdminEntry;
+import org.ehrbase.fhirbridge.ehr.opt.patientenaufenthaltcomposition.definition.FachlicheOrganisationseinheitCluster;
+import org.ehrbase.fhirbridge.ehr.opt.patientenaufenthaltcomposition.definition.StandortCluster;
+import org.ehrbase.fhirbridge.fhir.support.Encounters;
+import ca.uhn.fhir.rest.server.exceptions.UnprocessableEntityException;
+import org.hl7.fhir.r4.model.Encounter;
+import org.hl7.fhir.r4.model.Coding;
+import java.util.ArrayList;
+
+public class VersorgungsaufenthaltAdminEntryConverter extends EntryEntityConverter<Encounter, VersorgungsaufenthaltAdminEntry> {
+
+    private static final String FACH_ABTEILUNGS_SCHLUESSEL_CODE_SYSTEM = "https://www.medizininformatik-initiative.de/fhir/core/modul-fall/CodeSystem/Fachabteilungsschluessel";
+
+    @Override
+    protected VersorgungsaufenthaltAdminEntry convertInternal(Encounter encounter) {
+        VersorgungsaufenthaltAdminEntry versorgungsaufenthaltAdminEntry = new VersorgungsaufenthaltAdminEntry();
+
+        if(Encounters.isNotEmpty(encounter.getLocation())) {
+
+            Encounter.EncounterLocationComponent location = encounter.getLocation().get(0);
+
+            TimeConverter.convertEncounterLocationTime(location).ifPresent(versorgungsaufenthaltAdminEntry::setBeginnValue);
```

Comment:
This should be added to a generic class, if not otherwise possible TimeConverter should be never used directly in the specifc package to reduce [coupling](https://en.wikipedia.org/wiki/Coupling_(computer_programming)) . I would suggest adding a generic class EncounterToAdminEntryConverter that inherits EntryEntityConverter as an example see [MedicationStatementToObservationConverter](https://github.com/ehrbase/fhir-bridge/blob/develop/src/main/java/org/ehrbase/fhirbridge/ehr/converter/generic/MedicationStatementToObservationConverter.java).